services/QuillLMS/app/controllers/application_controller.rb

```diff
@@ -8,6 +8,7 @@ class ApplicationController < ActionController::Base
   GOOGLE_OR_CLEVER_JUST_SET = :google_or_clever_just_set
   KEEP_ME_SIGNED_IN = :keep_me_signed_in
   COMPREHENSION = 'comprehension'
+  EVIDENCE = 'evidence'
```

Comment:
Do we need to remove line 10 here with the `COMPREHENSION` definition?

Reply:
Good question - no, this has to stay until we change the webpack definition for the file names, which I think we are going to do as part of a code-name change at a later date.